tests/test_runlib.py

```diff
@@ -188,6 +188,36 @@ def test_base_path_is_parent_dir(self):
 
     os.chdir(self.test_dir)
 
+
+  def test_lstrip_paths_valid_prefix_directory(self):
+    lstrip_paths = "subdir/subsubdir/"
```

Comment:
In the prefix? The left strip would just fail silently in that instance as the test setup doesn't have such files. Should a file also be added then, which is explicitly unicode?

Reply:
It's okay to change the test case setup or write your own test cases.

Reply:
https://github.com/adityasaky/in-toto/commit/778aee7c7d40601967b98a0f00438f9f9b2fa049

I have this commit in my repo that adds such a test. The reason I haven't pushed to the branch here is because it requires me to explicitly declare the encoding. `tox` was complaining when I didn't, and I just wanted to check before I pushed it here.

Reply:
Declaring the encoding in that test module is fine. Please also check why tests are failing on appveyor.

Reply:
Whoops, didn't notice the Windows build failed, apologies.

Looking it up, I see that I need to add `encoding="utf-8"` in `open()` for Python 3 on Windows. However, `encoding` isn't valid in Python 2, so adding that will cause the Python 2 test to fail. Thoughts?

Reply:
Also, worth noting that the underlying system may support unicode file paths and still have this issue, as it seems to be an issue with file handling, and printing to stdout (boiling down to stdout's support for unicode).